system/CodeIgniter.php

```diff
@@ -1092,6 +1116,8 @@ protected function sendResponse()
      * @param int $code
      *
      * @deprecated 4.4.0 No longer Used. Moved to index.php.
+     *
+     * @return never
```

Comment:
The `never` is correct type for this method, but this method is special one for testing purpose.

Why don' t you change to `@return void` to pass PHPStan checks?
After all, this method will be removed in the future.